packages/legend-studio/src/components/viewer/Viewer.tsx

```diff
@@ -63,6 +63,8 @@ import {
   useApplicationStore,
 } from '@finos/legend-application';
 import type { LegendStudioConfig } from '../../application/LegendStudioConfig';
+import { GoEye } from 'react-icons/go';
```

Comment:
Do me a favor, let's put this in `Icon.tsx`, also find other places we do this and swap